### PR TITLE
Project Readability Improvements

### PR DESCRIPTION
**Pull request by gavingmiller**
Fixup of the Abuse Case Cheat Sheet to clarify language, fix horizontal scrolling on quote boxes, and remove unnecessary words/language.

### CODE REVIEW

**Comment by righettod** [comment]
Hello,
Can you merge all your PR (#170 #171 and #172) into a single one?
It facilitate the review and the tracking by the core team.
Thank you very much in advance 😃

**Comment by ThunderSon** [comment]
Hi Gavin 😄 Thank you for noticing the issues you handled. From reading the document, I noticed that there are still improvements that could be done grammar wise. I will try to point them out in the day or tomorrow, just letting you know. You might be able to handle them before I point them out

File: cheatsheets/Protect_FileUpload_Against_Malicious_File.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Into web applications, when we expect upload of working documents from users, we
 
 We use the term "malicious" here to refer to documents that embed *malicious code* that will be executed when another user (admin, back office operator...) will open the document with the associated application reader.
 
-Usually, when an application expect his user to upload a document, the application expect to receive a document for which the intended use will be for reading/printing/archiving. The document should not alter is content at opening time and should be in a final rendered state.
+Usually, when an application expects their user to upload a document, the application expects to receive a document for which the intended use will be for reading/printing/archiving. The document should not alter is content at opening time and should be in a final rendered state.
```
Comment (ThunderSon) [inline cheatsheets/Protect_FileUpload_Against_Malicious_File.md]: alter `its` content instead of `is` content in the last sentence

**Comment by righettod** [comment]
@gavingmiller Any update?

**Comment by righettod** [comment]
Thank a lot, @ThunderSon @mackowski can you take a look?

File: cheatsheets/Abuse_Case_Cheat_Sheet.md [hunk 2]
```diff
@@ -13,17 +13,15 @@ In order to build a secure application, from an pragmatic point of view, it is i
...
 **Important note about this Cheat Sheet:**
 
-```
-The main objective is to provide a pragmatic approach in order to allow a company or a project
-team to start building and handling the list of abuse cases and then customize the elements
-proposed to its context/culture in order to, finally, build its own method.
-
-This cheat sheet can be seen as a getting-started tutorial.
-```
+> The main objective is to provide a pragmatic approach in order to allow a company or a project
```
Comment (ThunderSon) [inline cheatsheets/Abuse_Case_Cheat_Sheet.md]: Just a small note for you to know. If you want the markdown generated from this code to be aligned as you expect it to be, you need to add new lines between every line. If you want we can discuss this on Slack.

File: cheatsheets/Abuse_Case_Cheat_Sheet.md [hunk 3]
```diff
@@ -113,15 +105,15 @@ It is important to take into account **Technical** and **Business** kind of abus
 *Example:*
 
 - Technical flagged abuse case: Add Cross Site Scripting injection into a comment input field.
-- Business flagged abuse case: Ability to modify  the price of an article in an online causing the user to pay a lower amount for the wanted article.
+- Business flagged abuse case: Ability to modify the price of an article in an online causing the user to pay a lower amount for the wanted article.
 
 ## When to define the list of Abuse Cases?
 
-On an agile project, the definition workshop must be made after the meeting in which User Stories are includen in a Sprint.
+On an Agile project, the definition workshop must be made after the meeting in which User Stories are included on a Sprint.
```
Comment (ThunderSon) [inline cheatsheets/Abuse_Case_Cheat_Sheet.md]: - `In Agile projects`. To follow how Waterfall is written, or the opposite, if that looks better to you.
- included **in** a sprint, and not **on**

File: cheatsheets/Abuse_Case_Cheat_Sheet.md [hunk 4]
```diff
@@ -113,15 +105,15 @@ It is important to take into account **Technical** and **Business** kind of abus
 *Example:*
 
 - Technical flagged abuse case: Add Cross Site Scripting injection into a comment input field.
-- Business flagged abuse case: Ability to modify  the price of an article in an online causing the user to pay a lower amount for the wanted article.
+- Business flagged abuse case: Ability to modify the price of an article in an online causing the user to pay a lower amount for the wanted article.
```
Comment (ThunderSon) [inline cheatsheets/Abuse_Case_Cheat_Sheet.md]: Change to something like: `Ability do modify the price of an article causing the user to pay a lower amount for the wanted article`.
I removed `in an online`. Unless you can fit a better sentence for it

**Comment by righettod** [comment]
Hello,

I can understand that the work implied when a PR is submitted is important and sometime need several phases of review.

All contributors and core team members follow the same review process. 

For example, even if I'm one of the project leaders, each time I submit a PR, all core team members made several reviews on my PR either on the content or the English (as I'm not an English native it's very important).

The goal is just to ensure the overall quality of the content because we are now a flagship project and this process is needed and mandatory.

All contributors and core team members work on this project on their spare time and it represents, for me/core team, a huge part of our spare time (for me it's around 1 hour every evening and around 1.5 days the weekend) but we accept that cost and the fact that sometimes it takes a while before a PR was merged or an issue was accepted/handled.

We handle all contributions proposal with the same approach and if, for you, this level of expectation become a barrier I can understand that.

However, as project leader, I cannot accept your last message to a member of my core team so I have decided to close the PR and reject your PR.

**Pull request closed without merging**